src/Components/FeelingRoom/FeelingRoom.js

```diff
@@ -10,10 +12,35 @@ class FeelingRoom extends Component {
     }
   }
   createFineRoom = () => {
-    //functions to animate go here
+    fineBackgroundAnimation()
```

Comment:
I wonder if we could refactor this to a for loop or something a bit more compact?
Not an issue, but certainly something to consider for the future.